roles/tpa_single_node/templates/manifests/vexination/indexer/Deployment.yaml.j2

```diff
@@ -9,79 +9,67 @@ metadata:
     app.kubernetes.io/instance: redhat-trusted-profile-analyzer
     app.kubernetes.io/version: 1.1.1
     app.kubernetes.io/part-of: trusted-profile-analyzer
-  annotations: 
+  annotations: null
 spec:
   replicas: 1
-
   strategy:
     type: Recreate
-
   selector:
     matchLabels:
       app.kubernetes.io/name: vexination-indexer
       app.kubernetes.io/component: vexination
       app.kubernetes.io/instance: redhat-trusted-profile-analyzer
-
   template:
     metadata:
       labels:
         app.kubernetes.io/name: vexination-indexer
         app.kubernetes.io/component: vexination
         app.kubernetes.io/instance: redhat-trusted-profile-analyzer
-
     spec:
       volumes:
         - name: data
           emptyDir: {}
-        - name: vex-indexer-storage-secret
+        - name: vexination-indexer-storage-secret
           secret:
             secretName: storage_secret
-        - name: vex-indexer-event-secret
+        - name: vexination-indexer-event-secret
           secret:
             secretName: event_secret
-
       containers:
         - name: service
-
           image: "{{ tpa_single_node_trustification_image }}"
           imagePullPolicy: IfNotPresent
-
           livenessProbe:
             initialDelaySeconds: 2
             httpGet:
               path: /health/live
               port: 9010
-          
           readinessProbe:
             initialDelaySeconds: 2
             httpGet:
               path: /health/ready
               port: 9010
-
           command: ["/trust"]
           args:
-            - "vexination"
-            - "indexer"
-            - "--stored-topic"
+            - vexination
+            - indexer
+            - --stored-topic
             - "{{ tpa_single_node_vexination_topic_stored }}"
-            - "--indexed-topic"
```

Comment:
Don't we need to have variables for thoses because they can change depending on the environment at least they are on S3.